features/support/ccd_support/et1_ccd_multiple_claimants.rb

```diff
@@ -147,6 +168,23 @@ def assert_secondary_claimant(claimants, representative, employment, respondents
           expect(secondary_claimants_left).to be_empty
         end
 
+        def download_primary_file
+          case_references = response['case_fields']['caseIdCollection'].first.dig('value', 'ethos_CaseReference')
+          created_case = ccd.caseworker_search_latest_by_ethos_case_reference(case_references, case_type_id: 'Manchester_Dev')
+
+          download = created_case['case_fields']['documentCollection'][0]['value']['uploadedDocument']['document_binary_url']
+          tempfile = Tempfile.new
+          File.open(tempfile, 'wb')  do |f|
```

Comment:
Should this not be using your helper method from above ccd_file_helper.rb ?

Reply:
doh, forgot to take this code out. thanks